client/blocks/reader-combined-card/style.scss

```diff
@@ -210,7 +210,6 @@
 	word-wrap: break-word;
 	position: relative;
 	margin-top: 4px;
-	white-space: nowrap;
```

Comment:
What's the rationale for removing this one?

Reply:
@bluefuton When a result data has a long text, this css property prevented it from wrapping and made the content to overflow as evident in the below screenshot (with this css property applied)

<img width="1167" alt="With whitespace nowrap applied" src="https://user-images.githubusercontent.com/11641900/91000425-ca731580-e5e6-11ea-949d-b490114acc04.png">

Also please refer the below screenshot for the same result data after removing this

<img width="1169" alt="Without whitespace nowrap" src="https://user-images.githubusercontent.com/11641900/91000495-ed052e80-e5e6-11ea-8ccc-a60d4a239558.png">